docs/source/conf.py

```diff
@@ -214,8 +214,9 @@
 nbsphinx_execute = 'always'
 nbsphinx_timeout = 600 # sphinx defaults each cell to 30 seconds so we need to override here
 
-inheritance_graph_attrs = dict(rankdir="LR", size='"1000, 333"',
-                               fontsize=30, labelfontsize=30, ratio='compress', dpi=960)
+inheritance_graph_attrs = dict(rankdir="TB", dpi=192.0, ranksep=0.02)
+inheritance_node_attrs = dict(fontsize=6, height=0.15, width=5, fixedsize=True)
```

Comment:
I wasn't able to figure out how to left-align the text. `label=r'\l'` created errors in rendering the docs.

Reply:
No worries, this is still great :)